src/main/scala/controllers/ApplicationController.scala

```diff
@@ -92,16 +96,28 @@ class ApplicationController @Inject()(actionHandler: ActionHandler, applications
   }
 
   def submit(id: ApplicationId) = Action.async { request =>
-    gatherApplicationDetails(id).map {
+    gatherApplicationDetails(id).flatMap {
       case Some((overview, form, opp)) =>
         val sectionErrors: Seq[SectionError] = form.sections.sortBy(_.sectionNumber).flatMap { fs =>
           overview.sections.find(_.sectionNumber == fs.sectionNumber) match {
             case None => Some(SectionError(fs, "Not started"))
             case Some(s) => checkSection(fs, s)
           }
         }
-        Ok(views.html.showApplicationForm(form, overview, opp, sectionErrors))
-      case None => NotFound
+        if(sectionErrors.isEmpty){
+          val emailto = "experiencederic@university.ac.uk"
+          val dtf = DateTimeFormat.forPattern("HH:mm:ss")
```

Comment:
This is something we should probably take up with the UX team. I don't want to hold up the merge of this PR, but it's definitely something to track.